client/packages/invoices/src/Prescriptions/DetailView/PrescriptionLineEdit/hooks/useDraftPrescriptionLines.tsx

```diff
@@ -78,6 +78,7 @@ export const useDraftPrescriptionLines = (
               invoiceLine,
               invoiceId,
               stockLine: batch,
+              adjustTotalNumberOfPacks: true,
```

Comment:
I went with this as it wasn't easy to map through the status of the invoice, could be added to outbound shipments too if we wanted?